nVolve/Shared/Models/InvolvedModel.swift

```diff
@@ -5,10 +5,10 @@
 //  Created by Luis on 11/12/24.
 //
 
-import SwiftUICore
+import SwiftUI
 
-struct InvolvedEvent: Decodable, Identifiable {
-    let id: String?
+struct InvolvedEvent: Codable, Identifiable, Equatable {
```

Comment:
Did we need to convert it to a `Codable` due so storing in local storage works like right here?
```swift
JSONEncoder().encode(favoriteEvents)
```